subjects/rt/README.md

```diff
@@ -0,0 +1,91 @@
+## rt
+
+There are two ways to render a 3d scene into a 2d image: `rasterization` that basically converts the shapes and geometric to pixels and applies calculations to obtain the color, shadows, refraction, etc... of those pixels. The other method is called `ray tracing` and consists in drawing each pixel with already its color, shadow, refraction, reflection, etc....
+
+Here we can see a visual difference between each method:
+
+![image.png](raytrace.png)
+
+![image.png](raytrace2.png)
+
+Imagine a camera pointing at a scene, and from that camera it is coming a bunch of rays tha bounce from object to object until it reaches the light source (lamp, sun, etc...). That is basically how a ray tracer works.
+
+In `ray tracing` each of these rays can be seen as a pixel in the image captured by the camera, and recursively the ray tracer will calculate from where the light comes from in that pixel, being able to give that pixel a color with some shadow aspect, some refraction aspect, and so on.
+
+To understand better how ray tracing works it is highly suggestive that you look online for this subject, as it can get quite complicated.
+
+### Objectives
+
+In this project it is supposed for you to implement the ray tracer method in order to be able to render a computer generated image containing a few objects.
+
+When building your ray tracer you have to take some points into consideration:
+
+- you need at least 4 simple objects: sphere, cube, plane and a cylinder.
+- your program must be able to change an object location before creating the image. (Example: render a sphere with its center on the point (1,1,1)).
+- you have to be able to look at the same scene from different angles by moving the camera/ point of view.
+- you also have to implement small light management, which includes: different brightness and shadows.
+
+As your ray tracer will probably be a bit slow to render high resolution scenes, you should make 4 .ppm images for the auditers to evaluate. The scenarios of these 4 images that you have to create consist of:
+
+- a scene with a sphere;
+- a scene with a plane and a cube with lower brightness than in the sphere image;
+- a scene with one of each of all the objects (one cube, one sphere, one cylinder and one plane);
+- a scene like the previous one, but with the camera in another position (thus generating the same image from a different perspective).
+
+All of the images should be in the format of 800x600. But while testing you can use smaller resolution images in order to not wait for a long time.
+
+Another aspect you should consider is that the auditer will have to use your ray tracer, so make it as usable and optimized as possible.
```

Comment:
Maybe we can ask them to create a small documentation on how to use their ray tracing API. You can give some points like in the `mini-framework` project, and also add to the audits